java/ingest/ingest-core/src/main/java/sleeper/ingest/job/status/IngestJobStatusStore.java

```diff
@@ -22,49 +22,112 @@
 import java.util.stream.Collectors;
 import java.util.stream.Stream;
 
+/**
+ * An interface for storing ingest job statuses.
```

Comment:
We can omit "An interface" here. See the Oracle style guide, "Class/interface/field descriptions can omit the subject and simply state the object": https://www.oracle.com/technical-resources/articles/java/javadoc-tool.html